src/App.tsx

```diff
@@ -1,339 +1,52 @@
-          photosphereID: "North",
-        },
-      },
-      {
-        pitch: -2,
-        yaw: -167,
-        tooltip: "Go to Entrance",
-        data: {
-          tag: "PhotosphereLink",
-          photosphereID: "Entrance",
-        },
-      },
-    ],
-  };
-
-  const entrance: Photosphere = {
-    id: "Entrance",
-    src: EntranceScene,
-    center: { x: 650, y: 930 },
-    hotspots: [
-      {
-        pitch: -2,
-        yaw: -4,
-        tooltip: "Go East",
-        data: {
-          tag: "PhotosphereLink",
-          photosphereID: "East",
-        },
-      },
-      {
-        pitch: -4,
-        yaw: -78,
-        tooltip: "Go South",
-        data: {
-          tag: "PhotosphereLink",
-          photosphereID: "South",
-        },
-      },
-    ],
-  };
-
-  const photospheres: Record<string, Photosphere> = {
-    [entrance.id]: entrance,
-    [south.id]: south,
-    [west.id]: west,
-    [north.id]: north,
-    [east.id]: east,
-  };
-
-  const map: NavMap = {
-    src: mapImage,
-    rotation: 0,
-    defaultZoom: 20,
-    hotspots: Object.values(photospheres).map((p) => {
-      return {
-        x: p.center.x,
-        y: p.center.y,
-        id: p.id,
-        color: "yellow",
-        tooltip: p.id,
-        data: { tag: "PhotosphereLink", photosphereID: p.id },
-      };
-    }),
-  };
-
-  const data: VFE = {
-    name: "prototypeElkIslandVFE",
-    map: map,
-    defaultPhotosphereID: south.id,
-    photospheres,
-  };
-
-  return <PhotosphereViewer vfe={data} />;
+import { useState } from "react";
+
+import CreateVFEForm from "./CreateVFE.tsx";
+import { VFE } from "./DataStructures.ts";
+import LandingPage from "./LandingPage.tsx";
+import PhotosphereEditor from "./PhotosphereEditor.tsx";
+import App from "./Prototype.tsx";
+
+// Main component acts as a main entry point for the application
+// Should decide what we are doing, going to LandingPage/Rendering VFE
+function AppRoot() {
+  // Decide state, should manage whether the VFE should be displayed or the LandingPage should be displayed
+  const [showApp, setShowApp] = useState(false);
+  const [showCreateVFEForm, setShowCreateVFEForm] = useState(false);
+  const [vfeData, setVFEData] = useState<VFE | null>(null);
+
+  //Create a function to set useState true
+  function handleLoadTestVFE() {
+    setShowApp(true);
+    setShowCreateVFEForm(false);
+  }
+
+  function handleCreateVFE() {
+    setShowCreateVFEForm(true);
+    setShowApp(false);
+  }
+
+  function loadCreatedVFE(data: VFE) {
+    setVFEData(data);
+    setShowApp(true);
+    setShowCreateVFEForm(false);
+  }
+
+  return (
+    <div>
+      {showCreateVFEForm ? (
+        <CreateVFEForm onCreateVFE={loadCreatedVFE} />
+      ) : vfeData && showApp ? (
+        <PhotosphereEditor vfe={vfeData} />
+      ) : !vfeData && showApp ? (
+        <App />
+      ) : (
+        <LandingPage
+          onLoadTestVFE={handleLoadTestVFE}
+          onCreateVFE={handleCreateVFE}
+        />
+      )}
```

Comment:
The nested ternaries make this hard to read.

It can abstracted into a local function that uses if statements and returns.

Reply:
Created the local function to handle the conditional checking